docs/Contributing/Understanding-host-vitals.md

```diff
@@ -662,7 +662,22 @@ SELECT
   '' AS vendor,
   0 AS last_opened_at,
   path AS installed_path
-FROM homebrew_packages;
+FROM homebrew_packages
+WHERE type = 'formula'
+UNION
+SELECT
+  name AS name,
+  version AS version,
+  '' AS bundle_identifier,
+  '' AS extension_id,
+  '' AS browser,
+  'homebrew_packages' AS source,
+  '' AS vendor,
+  0 AS last_opened_at,
+  path AS installed_path
+FROM homebrew_packages
+WHERE type = 'cask'
+AND NOT EXISTS (SELECT 1 FROM file WHERE file.path LIKE CONCAT(homebrew_packages.path, '/%%') AND file.path LIKE '/%.app%' LIMIT 1);
```

Comment:
Isn't this `file.path LIKE '/%.app%'` just matching something like `/Google Chrome.app`? (on the root path `/` only)

Reply:
I believe file.path is looking for a string match, which is why this is working:
```
osquery> SELECT path FROM file WHERE file.path LIKE CONCAT('/opt/homebrew/Caskroom/firefox', '/%%') AND file.path LIKE '/%.app%' LIMIT 1;
+-----------------------------------------------------+
| path                                                |
+-----------------------------------------------------+
| /opt/homebrew/Caskroom/firefox/126.0.1/Firefox.app/ |
+-----------------------------------------------------+
osquery> SELECT path FROM file WHERE file.path LIKE CONCAT('/opt/homebrew/Caskroom/ngrok', '/%%') AND file.path LIKE '/%.app%' LIMIT 1;
```

I think either should work, it seems clearer to use `%.app%` so i'll update